src/app/lib/constants/styles.js

```diff
@@ -5,6 +5,7 @@ import { injectGlobal } from 'styled-components';
 
 // Colours
 export const C_EBON = '#222222';
+export const C_GREY = '#4C4C4C';
```

Comment:
Will need to ask UX what the name of this colour is. I've just named it 'grey' for now.